examples/src/java/org/pantsbuild/example/README.md

```diff
@@ -211,6 +211,57 @@ Toolchain
 Pants uses [Ivy](http://ant.apache.org/ivy/) to resolve `jar` dependencies. To change how Pants
 resolves these, configure `resolve.ivy`.
 
+Starting release `1.4.0.dev26`, Pants added an option to pick [coursier](https://github.com/coursier/coursier)
+as the JVM 3rdparty resolver, with performance improvement being the main motivation. The goal is to retire ivy
+eventually. Example config to use coursier:
+
+    :::ini
+    # This will turn on coursier and turn off ivy.
+    [resolver]
+    resolver: coursier
+
+    [resolve.coursier]
+    # jvm option in case of large resolves
+    jvm_options: ['-Xmx4g', '-XX:MaxMetaspaceSize=256m']
+    # The opposite of resolve.ivy.soft_excludes
```

Comment:
Given that we have #4437 outstanding, is it too late to make soft_excludes the default for coursier?

Reply:
Ah I made behavior consistent with ivy in oss https://github.com/pantsbuild/pants/commit/35e711fbf1635e63ed550b77132b9cff3312ca13#diff-0b873245a3a9ee84a1f817502a221652R52

If you think we are not crossing release boundary yet (not sure if 1.4.0rc0 counts), I can just put a patch to flip the default.

Reply:
I'd be happy to cherry pick that in.